### PR TITLE
Reformatted date in tidied data set

### PR DESCRIPTION
**Pull request by lee-suddaby**
I'm a bit pedantic and have too much time on my hands. Do with this what you please. (there are also a few minor changes in the documentation)

### CODE REVIEW

**Comment by lee-suddaby** [comment]
Might have been a good idea to include @mine-cetinkaya-rundel when I opened this.

File: data-raw/dataprep.R [hunk 1]
```diff
@@ -11,8 +11,18 @@ squirrels <- read_csv(here::here("data-raw", "2018_Central_Park_Squirrel_Census_
   rename(
     long = x,
     lat = y
+  ) %>%
+  mutate(date = as.character(date)) %>%
+    mutate(
```
Comment (mine-cetinkaya-rundel) [inline data-raw/dataprep.R]: Can you fix the indentation here @lee-suddaby?

Comment (lee-suddaby) [reply data-raw/dataprep.R]: Should be looking better now @mine-cetinkaya-rundel

**Comment by mine-cetinkaya-rundel** [comment]
Thank you!

**Pull request merged**